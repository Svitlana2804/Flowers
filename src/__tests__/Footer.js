import React from "react";
import { render } from "@testing-library/react";
import Footer from "../composition/Footer/Footer";
// import { expect } from "vitest";
// import { describe } from 'vitest';

describe("Footer", () => {
  test("snapshot FOOTER", () => {
    const footer = render(<Footer />);
    expect(footer).toMatchSnapshot();
  });
});
