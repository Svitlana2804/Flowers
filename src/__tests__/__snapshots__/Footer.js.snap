// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot FOOTER 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer
        class="footer"
      >
        <div
          class="container"
        >
          <p>
            <a
              href="#"
            >
              <img
                alt="instagram"
                src="./public/icons/insta.png"
              />
            </a>
          </p>
          <p>
            Наша адреса: місто Черкаси, вул.Симоненка 
          </p>
          <p>
            Телефон: 
            <a
              href="tel:+380978888888"
            >
              +380978888888
            </a>
          </p>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer
      class="footer"
    >
      <div
        class="container"
      >
        <p>
          <a
            href="#"
          >
            <img
              alt="instagram"
              src="./public/icons/insta.png"
            />
          </a>
        </p>
        <p>
          Наша адреса: місто Черкаси, вул.Симоненка 
        </p>
        <p>
          Телефон: 
          <a
            href="tel:+380978888888"
          >
            +380978888888
          </a>
        </p>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
